meshtastic/interfaces/ble/receive_service.py

```
@@ -0,0 +1,714 @@
+        try:
+            iface._handle_from_radio(payload)
+        except DecodeError as exc:
+            logger.warning("Failed to parse FromRadio packet, discarding: %s", exc)
+            iface._read_retry_count = 0
+            return True
+        BLEReceiveRecoveryService._reset_recovery_after_stability(iface)
+        iface._read_retry_count = 0
+        return True
+
+    @staticmethod
+    def _handle_payload_read(
```

Ruff (0.15.5)

[warning] Too many return statements (9 > 6)

(PLR0911)